programs/monaco_protocol/src/instructions/order_request/create_order_request.rs

```diff
@@ -25,7 +28,9 @@ pub fn create_order_request(
     data: OrderRequestData,
 ) -> Result<u64> {
     let now: UnixTimestamp = current_timestamp();
-    validate_order_request(market, market_outcome, price_ladder, &data, now)?;
+    // unpack account optionals (works only for non-mut)
+    let price_ladder_account = price_ladder.as_ref().map(|v| v.clone().into_inner());
```

Comment:
NP: do you know how much compute this takes? I think price ladder can be quite big and doing a clone is maybe expensive

Reply:
deref used instead by @eoin-betdex advice